### PR TITLE
hipcc doc link

### PR DESCRIPTION
**Pull request by randyh62**
corrected link to HIPCC documentation

### CODE REVIEW

File: docs/what-is-rocm.rst [hunk 1]
```diff
@@ -121,7 +121,7 @@ Compilers
 
   "`AOMP <https://github.com/ROCm/aomp/>`_", "Scripted build of `LLVM <https://github.com/ROCm/llvm-project>`_ and supporting software"
   "`FLANG <https://github.com/ROCm/flang/>`_", "An out-of-tree Fortran compiler targeting LLVM"
-  "`hipCC <https://github.com/ROCm/HIPCC>`_ ", "Compiler driver utility that calls Clang or NVCC and passes the appropriate include and library options for the target compiler and HIP infrastructure"
+  "`hipCC <https://rocm.docs.amd.com/projects/HIPCC/en/latest/index.html>`_ ", "Compiler driver utility that calls Clang or NVCC and passes the appropriate include and library options for the target compiler and HIP infrastructure"
```
Comment (yhuiYH) [inline docs/what-is-rocm.rst:124]: Can you implement using the doc role, like the examples above in the ML&CV or Primitives section?  

https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role

**Review by yhuiYH — DISMISSED** [review]

File: docs/what-is-rocm.rst (hunk 1 above)
Comment (peterjunpark) [reply docs/what-is-rocm.rst:124]: I usually refer to https://github.com/ROCm/rocm-docs-core/blob/develop/src/rocm_docs/data/projects.yaml

Find the HIPCC key and use the same capitalization/spelling in the `:doc:` directive

Comment (neon60) [reply docs/what-is-rocm.rst:124]:
```suggestion
":doc:`hipCC <hipcc:index>`", "Compiler driver utility that calls Clang or NVCC and passes the appropriate include and library options for the target compiler and HIP infrastructure"
```

Comment (neon60) [reply docs/what-is-rocm.rst:124]: @randyh62 for me the suggested version worked.

Comment (neon60) [reply docs/what-is-rocm.rst:124]: @peterjunpark was faster.

File: docs/what-is-rocm.rst [hunk 2]
```diff
@@ -121,7 +121,7 @@ Compilers
 
   "`AOMP <https://github.com/ROCm/aomp/>`_", "Scripted build of `LLVM <https://github.com/ROCm/llvm-project>`_ and supporting software"
   "`FLANG <https://github.com/ROCm/flang/>`_", "An out-of-tree Fortran compiler targeting LLVM"
-  "`hipCC <https://github.com/ROCm/HIPCC>`_ ", "Compiler driver utility that calls Clang or NVCC and passes the appropriate include and library options for the target compiler and HIP infrastructure"
+":doc:`hipCC <hipcc:index>`", "Compiler driver utility that calls Clang or NVCC and passes the appropriate include and library options for the target compiler and HIP infrastructure"
```
Comment (neon60) [inline docs/what-is-rocm.rst:124]:
```suggestion
  ":doc:`hipCC <hipcc:index>` ", "Compiler driver utility that calls Clang or NVCC and passes the appropriate include and library options for the target compiler and HIP infrastructure"
```

Comment (neon60) [reply docs/what-is-rocm.rst:124]: @randyh62 I accidentally removed some spaces.

**Review by lpaoletti — APPROVED** [review]

**Review by lpaoletti — APPROVED** [review]

**Pull request merged**